doc/sphinx/conf.py.in

```diff
@@ -287,11 +288,10 @@ man_pages = [
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-  (master_doc, 'ExtensibleSimulationPackageforSoftMatterResearch', 'Extensible\
```

Comment:
Is there a reason for having the target names be inconsistent, that is lower case vs. camel case?

Reply:
we don't use latex anyway...

Reply:
The lowercase target name at line 277 is for man pages, which are usually case-insensitive. I'm not even sure we can generate man pages, I tried with Python2 and got a warning concerning unsupported tabular format in tables with rowspans (`"table cell spanning" not supported`). Couldn't try on Python3 because of the `PyUnicode_InternFromString` error in my environment.